README.md

```diff
@@ -1,57 +1,103 @@
 
-[npm-url]:https://npmjs.org/package/@moxy/{package-name}
-[downloads-image]:https://img.shields.io/npm/dm/@moxy/{package-name}.svg
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-circle
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-circle.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-circle.svg
+[build-status-url]:https://github.com/moxystudio/react-circle/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-circle/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-circle
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-circle/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-circle
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-circle.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-circle?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-circle.svg
 
-{package-description}
+A React component to render a circle with a partially or fully drawn stroke.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-circle
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+This package was implemented to tackle situations where you need to render a circle where only a percentage of the stroke is drawn (e.g. a loading indicator).
 
 ## Usage
 
-{package-usage-example}
+```js
+import React from 'react';
+import Circle from '@moxy/react-circle';
+
+const MyComponent = () => (
+    <div>
+        <Circle strokeWidth={ 2 } strokePercentage={ 0.7 } direction="clockwise" />
+    <div>
+);
+```
 
 ## API
 
-{package-api-description}
+The following props are available to customize `react-circle`'s behavior.
+
+#### strokeWidth
+
+Type: `number` | Required: `false` | Default: `1.5`
+
+The width of the circle's contour.
+
+#### strokePercentage
+
+Type: `number` | Required: `false` | Default: `1`
+
+The percentage of the circle's contour that is drawn.
+
+#### direction
+
+Type: `string` | Required: `false` | Default: `clockwise`
```

Comment:
You are saying here `Required: false`

Reply:
I'm not sure I understand what you mean here